### PR TITLE
[openslide] Port openslide.

### LINKED ISSUE #3660: Port Request: OpenSlide
OpenSlide is a C library that provides a simple interface to read whole-slide images (also known as virtual slides).

GitHub: https://github.com/openslide
Website: https://openslide.org/

### PR DESCRIPTION
**Pull request by onurtemizkan**
Fixes: #3660

### CODE REVIEW

File: ports/openslide/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+if(VCPKG_LIBRARY_LINKAGE STREQUAL "static")
+    file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/bin ${CURRENT_PACKAGES_DIR}/debug/bin)
+endif()
+
+file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include ${CURRENT_PACKAGES_DIR}/debug/share)
+
+vcpkg_copy_pdbs()
+
+file(INSTALL ${SOURCE_PATH}/LICENSE.txt DESTINATION ${CURRENT_PACKAGES_DIR}/share/openslide RENAME copyright)
```
Comment (NancyLi1013) [inline ports/openslide/portfile.cmake:43]: Could you please update this as
`file(INSTALL ${SOURCE_PATH}/LICENSE.txt DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)`?

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

**Comment by NancyLi1013** [comment]
Hi @onurtemizkan 
It seems that this port cannot work in current statue.

**Comment by onurtemizkan** [comment]
Hi @NancyLi1013, I have been distracted by other work after my first attempt on this port. But I'm planning to do some more work on this PR when I am able to. So, please keep it open until then.

**Comment by NancyLi1013** [comment]
@onurtemizkan 
Thanks for your update.

**Comment by NancyLi1013** [comment]
Hi @onurtemizkan 
Is work still being done for this PR?

**Comment by NancyLi1013** [comment]
Thanks for the PR; we're closing this for now since there's been no response. If you'd like to continue working on it, please reopen and ping us!

**Pull request closed without merging**